src/main/java/picard/util/BarcodeEditDistanceQuery.java

```diff
@@ -28,19 +28,19 @@
  */
 public class BarcodeEditDistanceQuery {
 
-    // list of barcodes (e.g. for dual-indexed barcodes)
+    /* list of barcodes (e.g. for dual-indexed barcodes) */
```

Comment:
ugh, sorry about this but you need two **s for javadoc comments
```
    /** list of barcodes (e.g. for dual-indexed barcodes) */
```

Reply:
right.